components/pontoon.jsx

```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+
+var enablePontoon = process.env.ENABLE_PONTOON === 'true';
```

Comment:
I just tested this- you should just be able to drop `process.env.ENABLE_PONTOON` in line 7 and get rid of this bit and it works as expected, just like the other env vars in this repo.